htdocs/admin/modules.php

```diff
@@ -685,7 +684,7 @@
         	}
         	else
         	{
-        		print '<a class="reposition" href="modules.php?id='.$objMod->numero.'&amp;module_position='.$module_position.'&amp;action=reset&amp;value=' . $modName . '&amp;mode=' . $mode . $param . '">';
+        		print '<a name="'.$modName.'" href="modules.php?id='.$objMod->numero.'&amp;module_position='.$module_position.'&amp;action=reset&amp;value=' . $modName . '&amp;mode=' . $mode . $param . '">';
```

Comment:
Using the # and name is not as efficient than using the class="reposition". Because with this method, the reposition is not done exactly at same place than before clicking. Why removing this ?